sdcm/logcollector.py

```diff
@@ -1202,6 +1203,9 @@ class JepsenLogCollector(LogCollector):
     cluster_dir_prefix = "jepsen-data"
 
     def collect_logs(self, local_search_path: Optional[str] = None) -> list[str]:
+        if 'jepsen' not in get_test_name():
```

Comment:
hmmm, I've just fixed some issue on a different place assuming think about job names.... 

it's hacky, and can come back biting

Reply:
see
https://github.com/scylladb/scylla-cluster-tests/pull/8114/commits/2eaaaa3f8f2ab64bf1500c5253fd23bc5d227646

Reply:
I was struggling to find a way to clearly determine if the executed test was a jepsen test. For now found only this option - to check the job name

Reply:
I would say the best way would be to have access to the test configuration.

And validating specific configuration is set.

But that should suffice for now

Reply:
let's see if @soyacz and @vponomaryov might have other ideas regarding this

Reply:
The main fix makes the `jepsen` commit be not really needed.
I would prefer the latter one to be removed.

Reply:
I was thinking that maybe skip would save some time, for the case when we try to archive remote jepsen logs that don't exist. But `if jepsen_archive := self.archive_log_remotely(..)` call ends immediately in this case, as in:
```
17:28:05  Start collect logs for cluster jepsen-data
17:28:05  Unable to archive log `jepsen-scylla' to `./jepsen-data.tar.gz'
17:28:05  There are no logs collected for jepsen-data
```
So we gain nothing here and probably should indeed skip this? @fruch

Reply:
Let's drop that part

The main issue is that the collection failed, 

There are two ways to avoid it, for now one is enough.

If it should still cause an issue, we'll get back to the whiteboard

Reply:
Removed the 2nd commit.